src/test/java/core/basesyntax/service/impl/DataConverterImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.validation.DataValidator;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DataConverterImplTest {
+    private DataValidator dataValidator = new DataValidator();
+    private DataConverter dataConverter = new DataConverterImpl(dataValidator);
```

Comment:
DataConverterImpl should be initialized in a setup method annotated with @BeforeEach. This way a fresh instance of the class is created for each test.